data/kata-collect-data.sh.in

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 #
-# Copyright (c) 2017-2019 Intel Corporation
+# Copyright (c) 2017-2018 Intel Corporation
```

Comment:
Have you been playing with that time machine again @devimc? :) This is strictly correct (it's a revert), but does look at bit confusing ;)